tests/errors.rs

```diff
@@ -28,7 +39,7 @@ fn test_error_parse() {
     assert_eq!(
         res.unwrap_err().to_string(),
         format!(
-            "invalid TOML value, did you mean to use a quoted string? at line 2 column 9 in {}",
+            "failed to parse datetime for key `error` at line 2 column 9 in {}",
```

Comment:
How does this belong to this PR?

Reply:
@matthiasbeyer It was weird when I ran test on my local environment, it was failed and I got the different error message, so I made this change to pass test. It seems CI has diff result. I reverted this change.